doc/dialects/saphana.md

```diff
@@ -0,0 +1,94 @@
+# SAP HANA SQL Dialect
+
+The SAP HANA SQL dialect allows you to reach [HANA](https://www.sap.com/products/hana.html) databases via Virtual Schemas.
+
+## JDBC Driver
+
+Download the latest version of [SAP HANA JDBC driver](https://search.maven.org/search?q=g:com.sap.cloud.db.jdbc%20AND%20a:ngdbc&core=gav).
+
+### Upload JDBC Driver to EXAOperation
+
+1. [Create a bucket in BucketFS](https://docs.exasol.com/administration/on-premise/bucketfs/create_new_bucket_in_bucketfs_service.htm) 
+1. Upload the driver to BucketFS
+
+## Connecting to SAP HANA
+
+1. Create schema
+    ```sql
+    CREATE SCHEMA ADAPTER;
+    ```
+2. Create Adapter Script
+
+    You can install the adapter script via the special SQL command `CREATE JAVA ADAPTER SCRIPT`
+    
+    ```sql
+     --/
+        CREATE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
+    %scriptclass com.exasol.adapter.RequestDispatcher;
+    %jar /buckets/bfsdefault/saphana/virtualschema-jdbc-adapter-dist-1.19.0.jar;
+    %jar /buckets/bfsdefault/saphana/ngdbc-2.4.56.jar;
+    /
+    ;
+    ```
+
+3. Create a connection
+    
+    ```sql
+    CREATE OR REPLACE CONNECTION SAPHANA_CONNECTION 
+    TO 'jdbc:sap://write.ip.address.here:port' 
+    USER 'SYSTEM' 
```

Comment:
Don't use administrative accounts in examples.

Reply:
fixed